### DISCUSSION TITLE
Welcome to ci_edit Discussions!

### DISCUSSION THREAD

**Discussion by dschuyler** [google/ci_edit#222] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other users and developers of ci_edit:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other users and developers.
  * Welcome others and be open-minded. 

  To get started, comment below with an introduction of yourself.

**Comment by saumeeld** [comment]
Thank you for making this :)

**Reply by saumeeld** [reply]
Am I able to configure the autocorrect highlighting color?
Both the cursor and highlighting are red by default for me.
